lib/datasource/remote/model/member_model.dart

```diff
@@ -1,29 +1,32 @@
-import 'package:equatable/equatable.dart';
+import 'package:flutter/foundation.dart';
 import 'package:seeds/datasource/remote/model/serialization_helpers.dart';
 import 'package:seeds/domain-shared/user_citizenship_status.dart';
 
-class MemberModel extends Equatable {
+class MemberModel {
   final String account;
   final String nickname;
   final String image;
+  final String status;
   final bool isSeedsUser;
-  final UserCitizenshipStatus? citizenshipStatus;
 
   const MemberModel({
     required this.account,
     required this.nickname,
     required this.image,
+    required this.status,
```

Comment:
Not sure why status is required now - what is the benefit of this, since some member models will be special accounts or organizations.

Reply:
Because I don't want to serialize and deserialize a enum field, I prefer save primitives types (status as String).
So the enum its a run time builded by a getter.

Organizations an any other not seeds account has a default unknown value for status.